sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -452,91 +453,103 @@ pi_result OCL(piEnqueueMemBufferMap)(
   return ret_err;
 }
 
-// TODO: Remove the 'OclPtr' extension used with the PI_APIs.
-// Forward calls to OpenCL RT.
+pi_result piPluginInit(pi_plugin *PluginInit) {
+  strcpy(PluginInit->PluginVersion, SupportedVersion);
+  int CompareVersions = strcmp(PluginInit->PiVersion, SupportedVersion);
+  if (CompareVersions < 0) {
```

Comment:
If this comparison fails we already changed the `PluginInit->PluginVersion`. Should we move the comparison before the version is set in the plugin?

Reply:
Is that commit not pushed yet as I cannot see a change in the code?

Reply:
It is their in https://github.com/intel/llvm/pull/843/commits/4ce1cc4f35a095c32cc001d0140e69eba5df0480.

Reply:
Thank you! :+1: